typescript/sdk/src/deploy/HyperlaneProxyFactoryDeployer.ts

```diff
@@ -48,14 +48,19 @@ export class HyperlaneProxyFactoryDeployer extends HyperlaneDeployer<
       this.factories,
     ) as (keyof ProxyFactoryFactories)[]) {
       const factory = await this.deployContract(chain, factoryName, []);
-      const artifact = {
-        name: proxyFactoryImplementations[factoryName],
-        address: await factory.implementation(),
-        constructorArguments: '',
-        isProxy: true,
-      };
-      await this.verifyContract(chain, artifact);
-      this.addVerificationArtifacts(chain, [artifact]);
+      try {
+        const artifact = {
+          name: proxyFactoryImplementations[factoryName],
+          address: await factory.implementation(),
+          constructorArguments: '',
+          isProxy: true,
+        };
+        await this.verifyContract(chain, artifact);
+        this.addVerificationArtifacts(chain, [artifact]);
+      } catch (error) {
+        this.logger.warn(`Failed to verify ${factoryName} on ${chain}:`, error);
+        // Continue deployment even if verification fails
```

Comment:
i think you wanted to only try-catch the `await this.verifyContract(chain, artifact);`

now if the verification fails we have no artifact to post-verify with